test/printf.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
Ohhhhhhhh wait a minute, my mistake. I forgot that now my code will run in native VM support, so I wasn't testing the interpreter at all...

Yes, in the interpreter I do see this properly fail before your fix.